docs/resources.md

```diff
@@ -607,6 +610,46 @@ the container image
 [gcr.io/cloud-builders//gcs-fetcher](https://github.com/GoogleCloudPlatform/cloud-builders/tree/master/gcs-fetcher)
 does not support configuring secrets.
 
+#### S3 Storage Resource
+
+S3 Storage resource points to
+[AWS S3 storage](https://aws.amazon.com/s3/) service.
+
+To create S3 storage resources, you will need a `Secret` with data containing your IAM Key and Secret so the cluster can authenticate with AWS. The below example references a Secret named `my-aws-creds-secret` for this purpose.
+
+```yaml
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineResource
+metadata:
+  name: s3-artifact
+spec:
+  type: storage
+  params:
+    - name: type
+      value: s3
+    - name: Bucket
```

Comment:
Suggest lowercasing these param names to match the gcs storage type.

Reply:
They might work if the objects are publicly accessible, that's how the GCS YAML tests work.